### PR TITLE
Fix issues founded in end game

### PR DESCRIPTION
**Pull request by Flanker32**
- Bump azure version in config files, fixes [#1797902](https://dev.azure.com/mseng/VSJava/_workitems/edit/1797902)
- Clear combo box when fill items, fixes [#1797587](https://dev.azure.com/mseng/VSJava/_workitems/edit/1797587)
- Always get latest app service status, fixes [#1797916](https://dev.azure.com/mseng/VSJava/_workitems/edit/1797916)
- Exclude unused dependency from dom4j, fixes [#1797591](https://dev.azure.com/mseng/VSJava/_workitems/edit/1797591), [#1797900](https://dev.azure.com/mseng/VSJava/_workitems/edit/1797900),[#1797913](https://dev.azure.com/mseng/VSJava/_workitems/edit/1797913),[#1797914](https://dev.azure.com/mseng/VSJava/_workitems/edit/1797914),[#1797592](https://dev.azure.com/mseng/VSJava/_workitems/edit/1797592)
- Wrapper web app deployment within AzureTaskManager and add empty check in AzureOperationsContext, fixes [#1797882](https://dev.azure.com/mseng/VSJava/_workitems/edit/1797882)

### CODE REVIEW

File: Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/operation/AzureOperationsContext.java [hunk 1]
```diff
@@ -58,7 +59,9 @@ public static Object execute(final AzureOperationRef operation, final OperationP
         try {
             return proceedable.proceed();
         } finally {
-            AzureOperationsContext.pop();
+            if (CollectionUtils.isNotEmpty(operations.get())) {
```
Comment (wangmingliang-ms) [inline Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/operation/AzureOperationsContext.java:62]: this should be impossible, when will this be empty?

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/AppServiceComboBox.java [hunk 2]
```diff
@@ -59,9 +59,12 @@ public synchronized void refreshItemsWithDefaultValue(@NotNull T defaultValue) {
         this.addItem(defaultValue);
         subscription = this.loadItemsAsync()
                            .subscribe(items -> DefaultLoader.getIdeHelper().invokeLater(() -> {
-                               items.forEach(this::addItem);
-                               this.resetDefaultValue(defaultValue);
-                               this.setLoading(false);
+                               synchronized (AppServiceComboBox.this) {
+                                   AppServiceComboBox.this.removeAllItems();
+                                   items.forEach(this::addItem);
+                                   this.resetDefaultValue(defaultValue);
```
Comment (shenqianjin) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/AppServiceComboBox.java:65]: is it enough if we just synchronized line 62 & 63 ？

**Review by wangmingliang-ms — APPROVED** [review]

File: Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/operation/AzureOperationsContext.java (hunk 1 above)
Comment (Flanker32) [reply Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/operation/AzureOperationsContext.java:62]: for #1797882, we found some actions will execute in same thread, which will clear the operations stack, so check whether operations is empty here

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/AppServiceComboBox.java (hunk 2 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/AppServiceComboBox.java:65]: Will refactor the code in later pr.

**Pull request merged**